arkenv-yaml/build.gradle.kts

```diff
@@ -0,0 +1,91 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    base
+    kotlin("jvm") version "1.3.72"
```

Comment:
Any reason for lower version?

Reply:
I just upgraded to the latest 1.3.* version. We can upgrade to 1.4 in some other branch 😄